src/db/migrations/20250513034953_job_type_updates/migration.sql

```diff
@@ -0,0 +1,28 @@
+-- Custom migration to handle CRITERIA_POLYGONS removal
+BEGIN;
+
+-- Delete JobResults linked to Jobs with type CRITERIA_POLYGONS
+DELETE FROM "JobResult" 
+WHERE "job_id" IN (SELECT "id" FROM "Job" WHERE "type" = 'CRITERIA_POLYGONS');
+
+-- Delete JobAssignments linked to Jobs with type CRITERIA_POLYGONS
+DELETE FROM "JobAssignment" 
+WHERE "job_id" IN (SELECT "id" FROM "Job" WHERE "type" = 'CRITERIA_POLYGONS');
+
+-- Delete JobRequests linked to Jobs with type CRITERIA_POLYGONS
+DELETE FROM "JobRequest" 
+WHERE "type" = 'CRITERIA_POLYGONS';
+
+-- Delete Jobs with type CRITERIA_POLYGONS
+DELETE FROM "Job" 
+WHERE "type" = 'CRITERIA_POLYGONS';
```

Comment:
should have setup delete cascades😁

Reply:
you're probably right. I was kinda joking, every DB designer I've talked to over the years has a different opinion on whether delete cascades are a good idea or not. I think DBAs that have run into an incident related to delete cascade are generally against them; they'd rather be explicit about deletes like this.